packages/backend/src/graphql/resolvers/peer.ts

```diff
@@ -58,17 +62,18 @@ export const getPeer: QueryResolvers<ApolloContext>['peer'] = async (
   return peerToGraphql(peer)
 }
 
-export const getPeerByAddressAndAsset: QueryResolvers<ApolloContext>['peerByAddressAndAsset'] =
+export const getPeerByAddressAndAsset: QueryResolvers<TenantedApolloContext>['peerByAddressAndAsset'] =
   async (parent, args, ctx): Promise<ResolversTypes['Peer'] | null> => {
     const peerService = await ctx.container.use('peerService')
     const peer = await peerService.getByDestinationAddress(
       args.staticIlpAddress,
+      ctx.isOperator ? undefined : ctx.tenant.id,
       args.assetId
     )
     return peer ? peerToGraphql(peer) : null
   }
 
-export const createPeer: MutationResolvers<ApolloContext>['createPeer'] =
+export const createPeer: MutationResolvers<ForTenantIdContext>['createPeer'] =
```

Comment:
The `ForTenantIdContext` does not seem to be leveraged for this resolver. An assertion for `ctx.forTenantId` being defined should be used in resolvers using that context in order to proceed.